### PR TITLE
Add support for launching selenium

### PR DESCRIPTION
**Pull request by jkrems**
Tested against `testium-driver-sync` using `testium_browser=chrome npm t`.

### CODE REVIEW

File: test/testium-core.js [hunk 1]
```diff
@@ -47,6 +47,9 @@ tap.test('Init against hello-world', function(t) {
         'Sets a _testium_ cookie');
     })
     .then(function() {
+      if (testium) {
+        testium.close();
+      }
       t.end();
     })
     .then(null, function(error) {
```
Comment (khoomeister) [inline test/testium-core.js]: The `.catch` here & the above changes can be DRYed into a helper.

File: lib/testium-core.js [hunk 2]
```diff
@@ -73,7 +85,31 @@ function initTestium() {
...
+    return new Bluebird(function(resolve, reject) {
+      var seleniumUrl = config.get('selenium.serverUrl');
+      debug('Verify selenium: ', seleniumUrl);
+      var req = http.get(seleniumUrl + '/status', function(res) {
+        debug('Selenium /status: ', res.statusCode);
+        resolve(procs);
+      });
+      req.on('error', function(error) {
+        var oldStack = error.stack;
+        oldStack = oldStack.substr(oldStack.indexOf('\n') + 1);
```
Comment (khoomeister) [inline lib/testium-core.js]: I'm curious to know why this is done - un momento per un commento?

**Comment by khoomeister** [comment]
:shipit:

File: test/testium-core.js (hunk 1 above)
Comment (jkrems) [inline test/testium-core.js]: Good point, actually `tap` accepted my PR to add promise support, so now they shouldn't be needed anymore. Will create a separate PR to clean that up.

File: lib/testium-core.js (hunk 2 above)
Comment (jkrems) [inline lib/testium-core.js]: Honest answer: This was blindly copied from testium. Created https://github.com/testiumjs/testium-core/issues/9 to look into it.

**Pull request merged**